DQMOffline/Trigger/python/SusyMonitoring_cff.py

```diff
@@ -2,6 +2,89 @@
 
 from DQMOffline.Trigger.RazorMonitor_cff import *
 
+
+from DQMOffline.Trigger.TopMonitor_cfi import hltTOPmonitoring
+
+
+#george
+#muon
+double_soft_muon_muonpt = hltTOPmonitoring.clone()
+double_soft_muon_muonpt.FolderName   = cms.string('HLT/SUSY/SOS/Muon/')
+# Selections
+double_soft_muon_muonpt.nmuons           = cms.uint32(2)
+double_soft_muon_muonpt.HTdefinition     = cms.string('pt>30 & abs(eta)<2.4')
+double_soft_muon_muonpt.HTcut            = cms.double(60)
+double_soft_muon_muonpt.metSelection     =cms.string('pt>150')
+double_soft_muon_muonpt.MHTdefinition    = cms.string('pt>30 & abs(eta)<2.4')
+double_soft_muon_muonpt.MHTcut           = cms.double(150)
+double_soft_muon_muonpt.invMassUppercut  = cms.double(50)
+double_soft_muon_muonpt.invMassLowercut  = cms.double(10)
+# Binning
+double_soft_muon_muonpt.histoPSet.muPtBinning      =cms.vdouble(0,2,5,7,10,12,15,17,20,25,30,50)
+double_soft_muon_muonpt.histoPSet.muPtBinning2D    =cms.vdouble(0,2,5,7,10,12,15,17,20,25,30,50)
+# Triggers
+double_soft_muon_muonpt.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_DoubleMu3_DZ_PFMET50_PFMHT60_v*')
+double_soft_muon_muonpt.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_PFMET140_PFMHT140_v*')
```

Comment:
No because we measure the muon leg efficiency. So we should be sure that the MET cut is in the plataeu. Normally it is reached at 130 GeV